packages/main/src/auth/useChannel.js

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+
+const useChannel = channel_id => {
+  const BASE_API_URL = "https://chat.zuri.chat";
+  const token = sessionStorage.getItem("token");
+  //const member_id = localStorage.getItem("user");
+  const org_id = localStorage.getItem("currentWorkspace");
+  const member = JSON.parse(sessionStorage.getItem("user")) || null;
+  const member_id = member.id;
```

Comment:
If it's null, how do you want to get the id?

Reply:
I'm no longer making use of the file, deleted it.